investpy/__init__.py

```diff
@@ -2370,12 +2559,14 @@ def get_etfs_overview(country, as_json=False):
     if unidecode.unidecode(country.lower()) not in get_etf_countries():
```

Comment:
All the changes done not related with `cross_currencies` should not be included since the previous code was properly formatted and the published changes are just modifications on how the data is tabulated or presented to the programmer. So **just the additions related to cross currencies data retrieval will be taken into consideration**, unless changes in any other function improves it.